Lab2.ipynb

```diff
@@ -0,0 +1,608 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Probability of background producing a signal of 2.50 or greater: 0.006209665325776159\n",
+      "Sigma associated with above probability 2.4999999999999987\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Problem 3 Part A\n",
+    "#Assume width of background distribution, X, equals +-15\n",
+    "#Assume signal strength, Y, equals 2.50\n",
+    "#Statistical Question: What is the probability of the background giving off a signal of 2.50 or greater?\n",
+    "\n",
+    "xlow = -15\n",
+    "xhigh = 15\n",
+    "xspace = 1500\n",
+    "\n",
+    "x = np.linspace(xlow,xhigh,xspace)\n",
+    "\n",
+    "gaus_dist = stats.norm.pdf(x,loc=0,scale=1)\n",
+    "\n",
+    "plt.step(x,gaus_dist)\n",
+    "plt.yscale('log')\n",
+    "plt.title('Background Shape, Zero-Mean Gaussian')\n",
+    "plt.xlabel('Value')\n",
+    "plt.ylabel('Probability')\n",
+    "plt.show()\n",
+    "\n",
+    "prob = stats.norm.cdf(.0250e2)\n",
+    "print('Probability of background producing a signal of 2.50 or greater:',1-prob)\n",
+    "\n",
+    "sig = stats.norm.ppf(prob)\n",
+    "print('Sigma associated with above probability',sig)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Part A Answer:\n",
+    "This is not a significant discovery because the sigma value from a signal of .0250e2 is only about 2.50, which does not meet the threshold to be considered significant, which would be a sigma value of 5 or more.\n",
```

Comment:
Consider including what the statistical question is for this scenario as well as the scenario in part B